string-manipulation-1/src/is-lower-cased.js

```diff
@@ -0,0 +1,10 @@
+/* exported isLowerCased */
+function isLowerCased(word) {
+  if (word.toLowerCase() === word) {
+    return true;
+  } else {
+    return false;
+  }
+  // eslint-disable-next-line no-unreachable
```

Comment:
Do not add any eslint rules without asking me first. If you need to add a rule to get past an error, that means you're doing something incorrect for the most part